lib/setup.js

```diff
@@ -42,23 +43,30 @@ import config from '../config'
 const {
   API_REQUEST_TIMEOUT,
   API_URL,
+  APP_TIMEOUT,
   COOKIE_DOMAIN,
   DEFAULT_CACHE_TIME,
   IP_BLACKLIST,
   NODE_ENV,
   OPENREDIS_URL,
   REQUEST_EXPIRE_MS,
   REQUEST_LIMIT,
+  SENTRY_PRIVATE_DSN,
   SESSION_COOKIE_KEY,
   SESSION_COOKIE_MAX_AGE,
-  SESSION_SECRET,
-  APP_TIMEOUT
+  SESSION_SECRET
 } = config
 
 export default function (app) {
   // Timeout middleware
   if (NODE_ENV === 'production') app.use(timeout(APP_TIMEOUT || '29s'))
 
+  // Error reporting
+  if (SENTRY_PRIVATE_DSN) {
```

Comment:
We have to gate this as a missing ENV var will send Force into a error -> forever restart loop